### ISSUE TITLE
pid manager gui needs more explicit names

### ISSUE THREAD

**Issue by lneuhaus** [pyrpl-fpga/pyrpl#298]

**Comment by SamuelDeleglise** [comment]
Not sure what you mean ? p->proportional i->integrator ?

**Comment by lneuhaus** [comment]
not sure either. The issue arose because Clement did not intuitively understand what the settings of p or i meant. I checked that the description is rather clear (in the tooltip), but probably it is not intuitive to hold the mouse cursor still on a AttributeWidget that one doesnt understand. Updating the tutorial / documentation will certainly help. Another improvement would be plotting the transfer function for each pid widget. what do you think about the plotting option?

**Comment by SamuelDeleglise** [comment]
Yes I had that in the back of my head for some time, however, I am not sure if it is not redundant with the lockbox transfer function plot that already exists.
